### PR TITLE
New version: SWI-Prolog.SWI-Prolog version 9.0.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Failed
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | SWI-Prolog | MasterGo 1.4.9, HTTP Toolkit 1.12.1, KDevelop    |
| Version     | 9.0.0     | 1.4.9, 1.12.1, 5.6.2 |
| Publisher   | SWI-Prolog   | 北京尽微致广信息技术有限公司, Tim Perry, KDE e.V.      |
| ProductCode |           | com.master.electron, d7e34dd1-b465-5f21-a0b6-9d800fa770d7, KDevelop    |
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/89572)

### CODE REVIEW

File: manifests/s/SWI-Prolog/SWI-Prolog/9.0.0/SWI-Prolog.SWI-Prolog.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+ShortDescription: A versatile implementation of the Prolog language
+Description: SWI-Prolog offers a comprehensive free Prolog environment. Since its start in 1987, SWI-Prolog development has been driven by the needs of real world applications. SWI-Prolog is widely used in research and education as well as commercial applications.
+Moniker: swipl
+Tags:
+- language
+- logic
+- programming
+- prolog
+# Agreements:
+ReleaseNotes: |-
```
Comment (russellbanks) [inline manifests/s/SWI-Prolog/SWI-Prolog/9.0.0/SWI-Prolog.SWI-Prolog.locale.en-US.yaml:27]: These release notes need updating

**Review by russellbanks — CHANGES_REQUESTED** [review]

File: manifests/s/SWI-Prolog/SWI-Prolog/9.0.0/SWI-Prolog.SWI-Prolog.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Commands:
+- swipl
+- swipl-ld
+FileExtensions:
+- pl
+- prolog
+- qlf
+ReleaseDate: 2022-06-21
+Installers:
+- Architecture: x86
```
Comment (SpecterShell) [inline manifests/s/SWI-Prolog/SWI-Prolog/9.0.0/SWI-Prolog.SWI-Prolog.installer.yaml:19]:
```suggestion
- Architecture: x64
```

File: manifests/s/SWI-Prolog/SWI-Prolog/9.0.0/SWI-Prolog.SWI-Prolog.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  InstallerUrl: https://www.swi-prolog.org/download/stable/bin/swipl-9.0.0-1.x64.exe
+  InstallerSha256: 195E249759CF12FC1FD8330E25A83F4701C3D5937ED3C09022DA4EF75633ADCF
+  ProductCode: swipl
+- Architecture: x64
```
Comment (SpecterShell) [inline manifests/s/SWI-Prolog/SWI-Prolog/9.0.0/SWI-Prolog.SWI-Prolog.installer.yaml:23]:
```suggestion
- Architecture: x86
```

File: manifests/s/SWI-Prolog/SWI-Prolog/9.0.0/SWI-Prolog.SWI-Prolog.locale.en-US.yaml (hunk 1 above)
Comment (mdanish-kh) [reply manifests/s/SWI-Prolog/SWI-Prolog/9.0.0/SWI-Prolog.SWI-Prolog.locale.en-US.yaml:27]: I guess they're too lengthy to be actually useful and we should probably get rid of them? To anyone who wants to read the whole changelog, the ReleaseNotesUrl may suffice.

File: manifests/s/SWI-Prolog/SWI-Prolog/9.0.0/SWI-Prolog.SWI-Prolog.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+
+  [Feb 14 2022]
+  - SANDBOX: SWISH issue#147: Allow for copy_term/4.
+  - FIXED: Allow retry from the body of => rules.
+  - ENHANCED: Avoid tracing inside spy/1, nospy/1 and related debug control predicates.
+  - FIXED: Issue#935: package manager: download location of git archives
+  - DOC: Remove obsolete reference of break/1 trace port
+
+  [Mar 22 2022]
+  - FIXED: Issue#963: possible undefined to_list/2 in message handling. Caused by wrong backport.
```
Comment (stephengillie) [inline manifests/s/SWI-Prolog/SWI-Prolog/9.0.0/SWI-Prolog.SWI-Prolog.locale.en-US.yaml:232]:
```suggestion
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**